libs/server/API/GarnetApiObjectCommands.cs

```diff
@@ -129,6 +129,10 @@ public GarnetStatus SortedSetRange(ArgSlice key, ArgSlice min, ArgSlice max, Sor
         public GarnetStatus SortedSetDifference(ArgSlice[] keys, out Dictionary<byte[], double> pairs)
             => storageSession.SortedSetDifference(keys, out pairs);
 
+        /// <inheritdoc />
+        public GarnetStatus SortedSetDifferenceStore(ArgSlice[] keys, ArgSlice destinationKey, out int count)
```

Comment:
For consistency with other similar commands, let's do: ```SortedSetDifferenceStore(byte[] key, ...)``` (see: SetDiffStore, SetIntersectStore etc.)

Reply:
If I do that, it won't be consistent with `SortedSetDifference`. Is that fine :-)

Reply:
SortedSetDifference doesn't have a dstKey...

Reply:
Ohh... Are you asking me to move the `destinationKey` to the first param and convert it to `byte[]` from `ArgSlice`?

I have previously seen that. I don't understand why we are passing it as `byte[]` instead of `ArgSlice` because we are converting `ArgSlice` to `byte[]` in the network layer and then converting `byte[]` to `ArgSlice` in the GarnetApi layer to create the transaction. To convert it back to `ArgSlice` we need additional memory, for that we are using scratchBufferManager. This looks inefficient to me when compared to directly passing it as `ArgSlice`. IMO, the existing places should be changed to use `ArgSlice`.

I don't prefer passing it as `byte[]` for this reason but if you like passing it as `byte[]` I will change it once you confirm it again :)

Reply:
Oh no no need for that, I just referred to the order of parameters.

Reply:
Changed the order of the param